style.css

```diff
@@ -454,3 +465,79 @@ textarea:focus {
   transform: scale(1.02);
   box-shadow: 0 0 10px #396df2;
 }
+
+/* MODAL_CONTENT_STYLE */
+.modal-container {
+  background-color: #fff;
+  border-radius: 16px;
+  border: 1px solid #9096a1;
+  padding: 30px;
+  position: absolute;
+  transform: scale(0.9);
+  box-sizing: border-box;
+  z-index: 5;
+}
```

Comment:
- [ ] [OPTIONAL]: I'm getting some weird flickering in your modal, I believe it's because of the project's cards animations. You should consider checking it out. You can see the bug here:


https://user-images.githubusercontent.com/66967274/146475366-e3d6b88c-1b32-436f-9952-5b89e3ba1199.mov

Reply:
I like the video on the comment, I would like to know that tool